backend/inventory-service/main.go

```
@@ -1 +1,298 @@
+	"github.com/louai60/e-commerce_project/backend/inventory-service/config"
+	"github.com/louai60/e-commerce_project/backend/inventory-service/handlers"
+	"github.com/louai60/e-commerce_project/backend/inventory-service/middleware"
+	pb "github.com/louai60/e-commerce_project/backend/inventory-service/proto"
+	"github.com/louai60/e-commerce_project/backend/inventory-service/repository/postgres"
+	"github.com/louai60/e-commerce_project/backend/inventory-service/service"
+)
+
+func main() {
+	// Initialize logger
+	logger := initLogger()
+	defer logger.Sync()
```

golangci-lint (1.64.8)

Error return value of `logger.Sync` is not checked

(errcheck)